idea_town/frontend/static-src/app/templates/not-found-page.js

```diff
@@ -0,0 +1,6 @@
+export default `
+  <section class="page" data-hook="not-found-page">
+    <h1>404 wut</h1>
+  </section>`
```

Comment:
Extra stray <code>`</code> here causes an "Unterminated template" error for me on build